recipes/gurkenlaeufer/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
```

Comment:
There are no patches, so less it more :)

```suggestion
from conan.tools.files import get, copy, rmdir
```